lmdeploy/turbomind/chat.py

```diff
@@ -1,22 +1,11 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 import dataclasses
 import os
-import os.path as osp
 import random
 
-os.environ['TM_LOG_LEVEL'] = 'ERROR'
-
+from .utils import get_gen_param
```

Comment:
can we use absolute import instead of relative import? IDEs now like vscode can modify corresponding codes if the file is moved to other places.